topology/Oak Ridge National Laboratory/ORNL/ORNL.yaml

```diff
@@ -0,0 +1,50 @@
+GroupDescription: ALICE USA Tier-2 site at LBNL
+GroupID: 536
+Production: true
+SupportCenter: USALICE
+Resources:
+  LBL_HPCS:
+    Active: true
+    ContactLists:
+      Administrative Contacts:
+        Primary:
+          ID: c0b37d66f9d39c4a37a83d20dfaf97f80d0c197e
+          Name: Peter Eby
+        Secondary:
+          ID: c3d417180e9c3738a32032eda0b9cf1252288870
+          Name: Jeff Porter
+      Resource Report Contact:
+        Primary:
+          ID: c3d417180e9c3738a32032eda0b9cf1252288870
+          Name: Jeff Porter
+        Secondary:
+          ID: 72150ad13dbc76a0f01443081df8e9172a06aa50
+          Name: Maarten Litmaath
+      Security Contact:
+        Primary:
+          ID: c0b37d66f9d39c4a37a83d20dfaf97f80d0c197e
+          Name: Peter Eby
+        Secondary:
+          ID: c3d417180e9c3738a32032eda0b9cf1252288870
+          Name: Jeff Porter
+    Description: ALICE USA Tier-2 site at LBNL
+    FQDN: osg.alice.ornl.gov
+    ID: 1053
+    Services:
+      CE:
+        Description: Compute Element
+        Details:
+          hidden: false
+    VOOwnership:
+      ALICE: 100
+    WLCGInformation:
+      APELNormalFactor: 11.75
+      AccountingName: US-LBNL-ALICE
```

Comment:
You might need to talk to someone at WLCG to figure out the right AccountingName but I'm guessing it's not "US-LBNL-ALICE".

Reply:
actually, the AccountingName is common for both sites in WLCG.   The project is run by LBNL under a single WLCG MoU.  So this one needs to stay the same.

Reply:
Interesting! OK, we'll keep it as is.